index.html

```diff
@@ -32,7 +32,105 @@
+            <span class="sr-only">Previous</span>
+          </a>
+          <a class="carousel-control-next" href="#carouselControls" role="button" data-slide="next">
+            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+            <span class="sr-only">Next</span>
+          </a>
         </div>
+      </header>
+
+      <nav class="navig container-fluid">
+        <ul class="lista row ">
+            <li
+                class="lista_home col-md-3">
+                <a
+                    href="#hero">HOME</a>
+            </li>
+            <li
+                class="lista_about col-md-3">
+                <a
+                    href="#function">ABOUT</a>
+            </li>
+            <li
+                class="lista_authors col-md-3">
+                <a
+                    href="#authors">AUTHORS</a>
+            </li>
+            <li
+                class="lista_extras col-md-3">
+                <a
+                    href="#extras">EXTRAS</a>
+            </li>
+        </ul>
+    </nav>
+    <main>
+    
+    <section class="functions--content">
+        
+        <h1 class="functions--content--header">Functions</h1>
+        
+        <div class="functions--content--container">
+
+          <div>
+
+              <h2 class="functions--content--header">
+                  1. Choose game
+              </h2>
+              <div class="test-img-container--1"></div>
+              <!-- <img src="./Ikony/kufer2.png" class="functions--img" alt="Game"> -->
+
+              <p class="functions--content--paragraph">
+                  First step towards sitting at a table with board game and players is to choose your desired game. Use the list of most popular games, or add one by your own.
+              </p>
+
+          </div>
+          
+          <div>
+
+              <h2 class="functions--content--header">
+                  2. Gather players
+              </h2>
+              
+              <div class="test-img-container--2"></div>
+
+              <!-- <img src="./Ikony/flgi.png" class="functions--img" alt="Players"> -->
+
+              <p class="functions--content--paragraph">
+                  Decide how many players you are looking for.
+              </p>
+
+          </div>
+
+          <div>
+
+              <h2 class="functions--content--header">
+                  3. Fix place and date
+              </h2>
+
+              <img src="./Ikony/zwój3.png" class="functions--img" alt="Place and date">
+              
+              <p class="functions--content--paragraph">
+                  Select location, date and time of your desired game session.
+              </p>
+
+          </div>
+          
+          <div>
+
+              <h2 class="functions--content--header">
+                  4. Reserve table
+              </h2>
+
+              <img src="./Ikony/piwo.png" class="functions--img" alt="Reserve table">
```

Comment:
it should be `icons/beer.png` use english please

Reply:
a matter of general team decisions and effort, I will pass it to others